storm-core/src/clj/org/apache/storm/ui/helpers.clj

```diff
@@ -46,197 +46,3 @@
   (fn [req]
```

Comment:
I prefer to leave it here, so when we move to new REST framework we can remove this code or the appropriate replacement code can be written.

I will add a comment here.